UNRELEASED.md

```diff
@@ -0,0 +1,4 @@
+**Changed**
+
+- `bpk-mixins`
+    - The `strong` badge theme has been added, which aligns to our mobile implementations and meets the needs of users.
```

Comment:
what does `aligns to our mobile implementations` mean?

Reply:
What I mean is that this theme color is already added on mobile for IOS and Android, so it is necessary to add it on the web as well.

Reply:
Have modified the sentence to make it clearer 😇